contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -756,17 +756,17 @@ GRANT SELECT ON information_schema_tsql.sequences TO PUBLIC;
  * SCHEMATA view
  */
 CREATE OR REPLACE VIEW information_schema_tsql.schemata AS
-	SELECT CAST(sys.db_name() AS sys.sysname) AS "CATALOG_NAME",
-	CAST(CASE WHEN np.nspname LIKE CONCAT(sys.db_name(),'%') THEN RIGHT(np.nspname, LENGTH(np.nspname) - LENGTH(sys.db_name()) - 1)
+	SELECT CAST(LOWER(sys.db_name()) AS sys.sysname) AS "CATALOG_NAME",
```

Comment:
Please verify whether this LOWER() for CATALOG_NAME column is needed or not?

Reply:
By this logic user queries like select * from information_schema.schemata where CATALOG_NAME = db_name() will stop working for case sensitive db names